Sample_v3/ConfigurationViewController.swift

```diff
@@ -40,7 +40,7 @@ class ConfigurationViewController: UITableViewController {
         case 2:
             tokenTypeSegmentedControl.selectedSegmentIndex = 2
             jwtTextField.isEnabled = false
-            if token != "" {
+            if !(token?.isEmpty ?? true) {
```

Comment:
Is this equal to `token?.isEmpty == false` or not?  🤔

Reply:
🤔 

1. Given we pass this condition only when token is not `""` which is the same as `!token.isEmpty` 
==>>  `(token != "") == !(token.isEmpty)`
1. When the token is nil, it means (for 99% of usecases when we check for string value and want to use it somewhere) automatically it's empty. So we replace immediately the value `isEmpty` with true, because it makes sense.
1. When we negate the `true`, we get obviously false, so it won't pass when either the **token is nil** or the **token is empty**.

This question has some valid point since I was while explaining this I got confused, you got me there for a second.